oak/server/module_invocation.cc

```diff
@@ -223,7 +221,7 @@ void ModuleInvocation::BlockingSendResponse() {
     auto callback = new std::function<void(bool)>(
         std::bind(&ModuleInvocation::SendResponse, this, std::placeholders::_1));
     stream_.Write(bb, options, callback);
-  } else if (!grpc_response.has_rsp_msg()) {
+  } else if (grpc_response.rsp_msg().empty()) {
```

Comment:
This is causing tests to fail now, I am guessing because an empty bytes field is indistinguishable from a default message. @daviddrysdale do we need to keep this distinction between last message with and without payload?

Reply:
I fixed it by assuming that only exactly one between `rsp_msg` and `status` is set. @ipetr0v is this compatible with the Rust implementation too? If so, should we make that a oneof, and / or explain in a comment in the proto message?